app/question.js

```diff
@@ -89,20 +85,17 @@ define(['app/messages', 'app/config'], function (messages,config) {
       var answer = items[currentItemIndex].answer;
 
       var difference = levenstein(input,answer);
-
+      
       if (difference == 0) {
-        messages.show( "Well done!", "success" );
+        messages.show( "Well done!", "success", config.constant("FEEDBACK_DELAY") );
         nextQuestion();
       } else if (difference <= (answer.length * config.constant("MARGIN_OF_ERROR"))) {
-        messages.show( "Almost there! Your answer: " + input + " - Expected answer: " + answer + " (" + difference + " letter(s) difference)", "warning");
+        messages.show( "Almost there! Your answer: " + input + " - Expected answer: " + answer + " (" + difference + " letter(s) difference)", "warning", config.constant("FEEDBACK_DELAY") );
```

Comment:
Possible enhancement: as you know the number of characters the sentences differ, you could determine if you need to letter or letters and simply use the correct form instead of letter(s).